Terminal.Gui/Views/TextField.cs

```diff
@@ -418,9 +418,9 @@ public TextField ()
     /// <summary>Gets or sets the foreground <see cref="Color"/> to use when rendering <see cref="Caption"/>.</summary>
     public Color CaptionColor { get; set; }
 
-    /// <summary>Get the <see cref="ContextMenu"/> for this view.</summary>
+    /// <summary>Get the Context Menu for this view.</summary>
     [CanBeNull]
-    public ContextMenuv2 ContextMenu { get; private set; }
+    public PopoverMenu ContextMenu { get; private set; }
```

Comment:
You wont replace these properties (`ContextMenu`) to `PopoverMenu `as well?

Reply:
I don't think so. 

PopoverMenu is a class that can be used for many things, including context menus. 

ContextMenu is a property name for the context menu of TextView. 

See?